rcon/cli.py

```diff
@@ -45,11 +45,6 @@ def run_stats_loop():
         sys.exit(1)
 
 
-@cli.command(name="record_server_stats_inception")
```

Comment:
Maybe just comment this out? This stuff is getting re-enabled at some point I assume.